src/config.yaml

```diff
@@ -0,0 +1,5 @@
+data_path: /home/sun00587/research/News_Locality_Polarization/poprox-recommender-locality/tests/request_data/test_request_data
```

Comment:
Not opposed to this. But if you want, we could keep the path variables in the script call and/or use relative paths (in the case of the output_dir).